packages/tailwindcss/src/utilities.test.ts

```diff
@@ -3142,10 +3142,6 @@ test('perspective', () => {
       --perspective-normal: 500px;
     }
 
-    .perspective-123 {
-      perspective: 123px;
-    }
-
```

Comment:
I think we should move the `perspective-123` example to the expect below where we explicitly test that it doesn't generate anything.